compiler/src/dotty/tools/dotc/transform/patmat/Space.scala

```diff
@@ -678,8 +669,8 @@ object SpaceEngine {
     rec(tp, Nil)
   }
 
-  extension (cls: Symbol)
-    /** A type is decomposable to children if it's sealed,
+  extension (tp: Type)
```

Comment:
This change changes binary compatibility of our public, yet private API. 
However, I don't think anyone would try to access any of this extension method

Reply:
We have enough problems and difficulties.  Let's not find ways to add new ones.  None of this is supported public API, and is subject to any change.